cumulus_library/cli.py

```diff
@@ -373,6 +389,10 @@ def run_cli(args: dict):
             elif args["action"] == "generate-sql":
                 for target in args["target"]:
                     runner.generate_study_sql(study_dict[target])
+
+            elif args["action"] == "generate-md":
```

Comment:
Ah - that's a nice analogy to `generate-sql` -- good job us on naming that one.

Reply:
yeah that worked out well, thanks past collective bikeshedding